util/rules_cleaner.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Actually, we've been trying to move to a python based CLI system, and ideally, this should be added to the qmk cli code, if possible. 

https://docs.qmk.fm/#/cli_development

If you don't think you can do that, that's fine, no worries.  But if you're able to, awesome!

Reply:
I'm not au fait with Python, so I'd have to take a look at this in the future after some learning.